pkg/mimirtool/commands/analyse.go

```diff
@@ -19,6 +19,7 @@ type AnalyzeCommand struct{}
 func (cmd *AnalyzeCommand) Register(app *kingpin.Application, envVars EnvVarNames) {
 	analyzeCmd := app.Command("analyze", "Run analysis against your Prometheus, Grafana, and Grafana Mimir to see which metrics are being used and exported.")
 	analyzeCmd.Flag("enable-experimental-functions", "If set, enables parsing experimental PromQL functions.").BoolVar(&parser.EnableExperimentalFunctions)
+	analyzeCmd.Flag("enable-experimental-duration-expressions", "If set, enables parsing experimental duration expressions.").BoolVar(&parser.ExperimentalDurationExpr)
```

Comment:
Not sure if this should be an option of not.

Reply:
If we're enabling it unconditionally in Mimir, we should enable it unconditionally here. Experimental functions are turned on or off per user so you want to behavior of `mimirtool` to match (hence the option).